src/components/Profile/Profile.component.js

```diff
@@ -1,25 +1,32 @@
 import React, {useEffect} from 'react';
 import {Avatar, Content, History, UserInfo, Wrapper} from "./Profile.style";
 import {useDispatch, useSelector} from "react-redux";
-import {getAsyncArticles} from "../../redux/articlesReducer";
 import {closeFilters} from "../../redux/filterReducer";
+import {VscSignOut} from "react-icons/all";
+import Button from "../../common/Buttons/Button.component";
```

Comment:
bring changes to webpack
and get possibility to use "src/components/Button/Button.component"